diff -Nur old/Documentation/rc_updatedd.dyndns new/Documentation/rc_updatedd.dyndns
--- old/Documentation/rc_updatedd.dyndns	2003-02-17 14:39:10.000000000 +0000
+++ new/Documentation/rc_updatedd.dyndns	2003-09-25 21:34:26.000000000 +0000
@@ -14,7 +14,7 @@
 SERVICE="dyndns"
 
 FLAGS="-S $SERVICE -n $HOSTNAMES -u $LOGIN -s $SYSTEM -y"
-UPDATEDD="/usr/local/sbin/updatedd"
+UPDATEDD="/usr/sbin/updatedd"
 
 
 if [ -x $DAEMON ]; then
diff -Nur old/Documentation/rc_updatedd.hn new/Documentation/rc_updatedd.hn
--- old/Documentation/rc_updatedd.hn	2003-02-17 14:38:34.000000000 +0000
+++ new/Documentation/rc_updatedd.hn	2003-09-25 21:34:33.000000000 +0000
@@ -12,7 +12,7 @@
 INTERFACE="ppp0"
 
 FLAGS="-S $SERVICE -i $INTERFACE -u $LOGIN -y"
-UPDATEDD="/usr/local/sbin/updatedd"
+UPDATEDD="/usr/sbin/updatedd"
 
 
 if [ -x $DAEMON ]; then
diff -Nur old/Documentation/rc_updatedd.noip new/Documentation/rc_updatedd.noip
--- old/Documentation/rc_updatedd.noip	2003-02-17 14:44:09.000000000 +0000
+++ new/Documentation/rc_updatedd.noip	2003-09-25 21:34:39.000000000 +0000
@@ -15,7 +15,7 @@
 SERVICE="noip"
 
 FLAGS="-S $SERVICE -n $HOSTNAME -e $EMAIL -p $PASS -i $INTERFACE -y"
-UPDATEDD="/usr/local/sbin/updatedd"
+UPDATEDD="/usr/sbin/updatedd"
 
 
 if [ -x $DAEMON ]; then
diff -Nur old/Documentation/rc_updatedd.ods new/Documentation/rc_updatedd.ods
--- old/Documentation/rc_updatedd.ods	2003-02-17 14:46:54.000000000 +0000
+++ new/Documentation/rc_updatedd.ods	2003-09-25 21:34:45.000000000 +0000
@@ -14,7 +14,7 @@
 SERVICE="ods"
 
 FLAGS="-S $SERVICE -n $HOSTNAME -i $INTERFACE -u $LOGIN -y"
-UPDATEDD="/usr/local/sbin/updatedd"
+UPDATEDD="/usr/sbin/updatedd"
 
 
 if [ -x $DAEMON ]; then
diff -Nur old/Documentation/rc_updatedd.ovh new/Documentation/rc_updatedd.ovh
--- old/Documentation/rc_updatedd.ovh	2003-02-17 14:50:22.000000000 +0000
+++ new/Documentation/rc_updatedd.ovh	2003-09-25 21:34:50.000000000 +0000
@@ -14,7 +14,7 @@
 SERVICE="ovh"
 
 FLAGS="-S $SERVICE -i $HOSTNAME -i $INTERFACE -u $LOGIN -y"
-UPDATEDD="/usr/local/sbin/updatedd"
+UPDATEDD="/usr/sbin/updatedd"
 
 
 if [ -x $DAEMON ]; then
